odbc/src/api/functions.rs

```diff
@@ -1211,6 +1226,20 @@ fn sql_free_handle(handle_type: HandleType, handle: *mut MongoHandle) -> Result<
                     .as_connection()
                     .ok_or(ODBCError::InvalidHandleType(HANDLE_MUST_BE_CONN_ERROR))?
             };
+            // We must take the connection attributes in the mongo_connection and put them back
+            // in the connection attributes for the conn
+            if conn.has_mongo_connection() {
```

Comment:
```suggestion
```

Reply:
Sorry this is two diffs because github doesn't seem to let you edit around your own comment lol